src/utils/mail/index.js

```diff
@@ -0,0 +1,83 @@
+//@ts-check
+import { compile } from 'ejs';
+import { readFileSync, ReadStream } from 'fs';
+import { minify } from 'html-minifier';
+import { createTransport } from 'nodemailer';
+import { join } from 'path';
+
+import { configs } from '../../configs';
+
+const rootPath = join(__dirname, '..', '..', '..', 'templates/');
+
+const templates = {
+    'general-template': 'general-template.ejs',
+    'access-granted': 'access-granted.ejs',
+};
+
+const opts = {
+    pool: true,
+    host: configs.smtpHost,
+    secure: false,
+    port: configs.smtpPort,
+    auth: {
+        user: configs.smtpEmail,
+        pass: configs.smtpPass,
+    },
+    tls: {
+        rejectUnauthorized: false,
+    },
+};
+
+//@ts-ignore
+const transporter = createTransport(opts);
+
+/**
+ * Send email to given email(s)
+ *
+ * @param {{to: string | string[], subject: string, html: string, attachments?: [{filename: string, content: ReadStream}]}} options
+ * @returns
+ */
+export const sendEmail = options => {
+    const { to, subject, html, attachments } = options;
```

Comment:
Add cc as an optional parameter. cc could be more than one mail.

Reply:
We are doing this programmatically we do not need a cc. Refer to comment above

Reply:
Most organisation use cc for people who are not required to take action to an email but should see the information. Maybe when you send an invite to all teachers. You can copy the board management to get aware of the invite.

Reply:
Cool done